[JIT] Fix module interface test

jamesr66a opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#39592 [JIT] Fix module interface test**

Closes https://github.com/pytorch/pytorch/issues/39523

Differential Revision: [D21909659](https://our.internmc.facebook.com/intern/diff/D21909659)

Commit:
[JIT] Fix module interface test

[ghstack-poisoned]

Commit:
Update on "[JIT] Fix module interface test"


Closes https://github.com/pytorch/pytorch/issues/39523

Differential Revision: [D21909659](https://our.internmc.facebook.com/intern/diff/D21909659)

[ghstack-poisoned]

wanchaol reviewed (approved):
are we getting rid of python 3.5 ci already? I remembered last time I did this in https://github.com/pytorch/pytorch/pull/35417 and it get reverted later.

wanchaol commented on test/test_jit.py:
```diff
@@ -25,6 +25,7 @@
 from jit.test_functional_blocks import TestFunctionalBlocks  # noqa: F401
 from jit.test_torchbind import TestTorchbind  # noqa: F401
 from jit.test_op_normalization import TestOpNormalization  # noqa: F401
+from jit.test_module_interface import TestModuleInterface # noqa: F401
```
please fix the lint before merging

Commit:
Update on "[JIT] Fix module interface test"


Closes https://github.com/pytorch/pytorch/issues/39523

Differential Revision: [D21909659](https://our.internmc.facebook.com/intern/diff/D21909659)

[ghstack-poisoned]
